File: python/MaxElementTilln'th.py
import sys 

def max(arr,n,min):
    for i in range(0,n):
        if min < arr[i]:
            min=arr[i]
            print(min)

        else:
            print(min)

arr = [1,2,3,4,1,10,22,2002,12,2001]
max(arr,len(arr),sys.minint)
            

File: java/MaxElementTilln'th.java
public class Algorithms{
    public static int main(){
        int arr[] = {1,2,3,4,1,10,22,2002,12,2001}; 
        int n = arr.length;

        int min = Integer.MIN_VALUE;
        for(int i=0;i<n;i++){
            if(min < arr[i]){
                min=arr[i];
                System.out.print(min);
            }else{
                System.out.print(min);
            }
            System.out.print(" ");
        }
    }
}